SNMPv3/README.md

```diff
@@ -0,0 +1,28 @@
+# SNMPv3 POC
```

Comment:
Rename POC, even as main topic in repo

**Network switch monitoring using SNMPv3**

Reply:
Folder name also needs to reflect this .. have folder name as NetworkSwitchMonitoring

Reply:
done..